packages/fscomponents/src/components/Form/Form.tsx

```diff
@@ -89,8 +110,17 @@ export class Form extends PureComponent<FormProps> {
       templates
     } = this.props;
 
+    Object.keys(fieldsOptions).forEach(path => {
```

Comment:
We should probably do this and compute the stylesheet in `getDerivedStateFromProps` to avoid re-computing this every render.